app/metrics.yaml

```diff
@@ -539,3 +539,16 @@ custom_tab:
     notification_emails:
     - fenix-core@mozilla.com
     expires: "2020-03-01"
+
+preferences:
+  preference_toggled:
+    type: event
+    description: >
```

Comment:
The extras need to be documented, you can look for an example here. 
https://github.com/mozilla-mobile/fenix/blob/master/app/metrics.yaml#L42

In the documentation also put all the available options